src/frontends/tests/onnx/onnx_import.in.cpp

```diff
@@ -5179,3 +5179,32 @@ NGRAPH_TEST(${BACKEND_NAME}, onnx_grid_sample) {
 
     test_case.run();
 }
+
+NGRAPH_TEST(${BACKEND_NAME}, onnx_model_layer_norm) {
+    const auto function = onnx_import::import_onnx_model(file_util::path_join(SERIALIZED_ZOO, "onnx/layer_norm.onnx"));
+
+    auto test_case = test::TestCase(function, s_device);
+    std::vector<float> a = {
```

Comment:
Both variables `a` and `b` don't mean much so please rename them or pass the values directly to the calls of `add_input` and `add_expected_output`.